<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 54px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 59px;
  margin-top: -59px;
}

.section h2 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h3 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h4 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h5 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h6 {
  padding-top: 59px;
  margin-top: -59px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">XIAO Project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="meeting.html">Meeting</a>
</li>
<li>
  <a href="Note.html">Note</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<h1 style="color:grey;font-family:arial; position:relative; top:15px;text-align:center;">
XIAO<img src="website_materials/research_website_logo.png"  style="float:right;width:200px;height:150px;">
</h1>
<p><br></p>
<hr />
<pre><code>Read in reads count of protein coding genes of the 1st batch mouse mRNA-Seq data. 
the count data were generated by htseq-count and were stored in ../data/Shuo_Xiao_RNAseq_mouse_batch1
Then use EdgeR to analyze differentiall expressed genes</code></pre>
<pre><code>## Loading required package: limma</code></pre>
<div id="read-in-count-of-each-protein-coding-gene-short-examples-shown-below" class="section level2">
<h2>Read in count of each protein coding gene, short examples shown below</h2>
<table>
<thead>
<tr class="header">
<th align="left">genename</th>
<th align="right">ART_1</th>
<th align="right">ART_2</th>
<th align="right">ART_3</th>
<th align="right">ART_4</th>
<th align="right">ART_5</th>
<th align="right">ART_6</th>
<th align="right">ART_7</th>
<th align="right">ART_8</th>
<th align="right">ART_9</th>
<th align="right">ART_10</th>
<th align="right">ART_11</th>
<th align="right">ART_12</th>
<th align="right">ART_13</th>
<th align="right">ART_14</th>
<th align="right">ART_15</th>
<th align="right">ART_16</th>
<th align="right">ART_17</th>
<th align="right">ART_18</th>
<th align="right">ART_19</th>
<th align="right">ART_20</th>
<th align="right">ART_21</th>
<th align="right">ART_22</th>
<th align="right">ART_23</th>
<th align="right">ART_24</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">0610007C21Rik</td>
<td align="right">1255</td>
<td align="right">1245</td>
<td align="right">1187</td>
<td align="right">1207</td>
<td align="right">1233</td>
<td align="right">736</td>
<td align="right">553</td>
<td align="right">851</td>
<td align="right">1397</td>
<td align="right">1528</td>
<td align="right">1464</td>
<td align="right">1061</td>
<td align="right">1388</td>
<td align="right">1020</td>
<td align="right">940</td>
<td align="right">1022</td>
<td align="right">557</td>
<td align="right">622</td>
<td align="right">732</td>
<td align="right">716</td>
<td align="right">1009</td>
<td align="right">1068</td>
<td align="right">1278</td>
<td align="right">1416</td>
</tr>
<tr class="even">
<td align="left">0610007L01Rik</td>
<td align="right">1621</td>
<td align="right">1463</td>
<td align="right">1388</td>
<td align="right">1441</td>
<td align="right">6498</td>
<td align="right">2445</td>
<td align="right">3786</td>
<td align="right">3009</td>
<td align="right">1630</td>
<td align="right">1374</td>
<td align="right">1578</td>
<td align="right">1813</td>
<td align="right">1874</td>
<td align="right">1430</td>
<td align="right">1316</td>
<td align="right">1343</td>
<td align="right">3835</td>
<td align="right">4040</td>
<td align="right">4563</td>
<td align="right">5191</td>
<td align="right">1648</td>
<td align="right">1723</td>
<td align="right">1782</td>
<td align="right">1999</td>
</tr>
<tr class="odd">
<td align="left">0610007P08Rik</td>
<td align="right">1146</td>
<td align="right">979</td>
<td align="right">1021</td>
<td align="right">1273</td>
<td align="right">1866</td>
<td align="right">1115</td>
<td align="right">1143</td>
<td align="right">1193</td>
<td align="right">1192</td>
<td align="right">826</td>
<td align="right">1226</td>
<td align="right">1221</td>
<td align="right">1162</td>
<td align="right">936</td>
<td align="right">957</td>
<td align="right">864</td>
<td align="right">1185</td>
<td align="right">1146</td>
<td align="right">1600</td>
<td align="right">1593</td>
<td align="right">1161</td>
<td align="right">1397</td>
<td align="right">1405</td>
<td align="right">1552</td>
</tr>
<tr class="even">
<td align="left">0610007P14Rik</td>
<td align="right">494</td>
<td align="right">443</td>
<td align="right">421</td>
<td align="right">454</td>
<td align="right">804</td>
<td align="right">470</td>
<td align="right">471</td>
<td align="right">455</td>
<td align="right">700</td>
<td align="right">640</td>
<td align="right">725</td>
<td align="right">620</td>
<td align="right">607</td>
<td align="right">447</td>
<td align="right">436</td>
<td align="right">549</td>
<td align="right">709</td>
<td align="right">586</td>
<td align="right">741</td>
<td align="right">663</td>
<td align="right">621</td>
<td align="right">723</td>
<td align="right">740</td>
<td align="right">785</td>
</tr>
<tr class="odd">
<td align="left">0610007P22Rik</td>
<td align="right">726</td>
<td align="right">564</td>
<td align="right">512</td>
<td align="right">555</td>
<td align="right">1698</td>
<td align="right">651</td>
<td align="right">862</td>
<td align="right">685</td>
<td align="right">553</td>
<td align="right">561</td>
<td align="right">565</td>
<td align="right">538</td>
<td align="right">654</td>
<td align="right">465</td>
<td align="right">447</td>
<td align="right">514</td>
<td align="right">939</td>
<td align="right">1013</td>
<td align="right">1047</td>
<td align="right">1271</td>
<td align="right">559</td>
<td align="right">571</td>
<td align="right">672</td>
<td align="right">646</td>
</tr>
<tr class="even">
<td align="left">0610009B22Rik</td>
<td align="right">303</td>
<td align="right">221</td>
<td align="right">194</td>
<td align="right">235</td>
<td align="right">223</td>
<td align="right">140</td>
<td align="right">133</td>
<td align="right">137</td>
<td align="right">253</td>
<td align="right">309</td>
<td align="right">319</td>
<td align="right">234</td>
<td align="right">267</td>
<td align="right">233</td>
<td align="right">195</td>
<td align="right">245</td>
<td align="right">191</td>
<td align="right">185</td>
<td align="right">180</td>
<td align="right">208</td>
<td align="right">247</td>
<td align="right">272</td>
<td align="right">281</td>
<td align="right">322</td>
</tr>
</tbody>
</table>
<pre class="r"><code>fwrite(count, &quot;./170305_read_count_table_for_protein_coding_genes.txt&quot;, col.names = TRUE, sep = &quot;\t&quot;, row.names = FALSE, quote = FALSE)</code></pre>
<ul>
<li><a href="170305_read_count_table_for_protein_coding_genes.txt"><strong>Click here for read count information for protein coding genes</strong></a></li>
</ul>
</div>
<div id="use-edger-to-call-differentially-regulated-genes" class="section level2">
<h2>use EdgeR to call differentially regulated genes</h2>
<pre><code>## Loading required package: splines</code></pre>
</div>
<div id="number-of-genes-up--and-down-regulated-at-qvalue-0.05" class="section level2">
<h2>Number of genes up- and down-regulated at qvalue &lt; 0.05</h2>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">up</th>
<th align="right">down</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>E2_PBS</td>
<td align="right">4940</td>
<td align="right">5068</td>
</tr>
<tr class="even">
<td>Progesterone_PBS</td>
<td align="right">2971</td>
<td align="right">2003</td>
</tr>
<tr class="odd">
<td>E2_DOX</td>
<td align="right">5508</td>
<td align="right">6153</td>
</tr>
<tr class="even">
<td>Progesterone_DOX</td>
<td align="right">2420</td>
<td align="right">1800</td>
</tr>
<tr class="odd">
<td>E2_DOX_specific</td>
<td align="right">302</td>
<td align="right">1212</td>
</tr>
<tr class="even">
<td>Progesterone_DOX_specific</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
</div>
<div id="number-of-genes-up--and-down-regulated-at-qvalue-0.01" class="section level2">
<h2>Number of genes up- and down-regulated at qvalue &lt; 0.01</h2>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">up</th>
<th align="right">down</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>E2_PBS</td>
<td align="right">4350</td>
<td align="right">4140</td>
</tr>
<tr class="even">
<td>Progesterone_PBS</td>
<td align="right">2059</td>
<td align="right">1263</td>
</tr>
<tr class="odd">
<td>E2_DOX</td>
<td align="right">4966</td>
<td align="right">5302</td>
</tr>
<tr class="even">
<td>Progesterone_DOX</td>
<td align="right">1541</td>
<td align="right">1107</td>
</tr>
<tr class="odd">
<td>E2_DOX_specific</td>
<td align="right">111</td>
<td align="right">703</td>
</tr>
<tr class="even">
<td>Progesterone_DOX_specific</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
</div>
<div id="number-of-genes-up--and-down-regulated-at-qvalue-0.001" class="section level2">
<h2>Number of genes up- and down-regulated at qvalue &lt; 0.001</h2>
<table>
<thead>
<tr class="header">
<th></th>
<th align="right">up</th>
<th align="right">down</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>E2_PBS</td>
<td align="right">3763</td>
<td align="right">3198</td>
</tr>
<tr class="even">
<td>Progesterone_PBS</td>
<td align="right">1235</td>
<td align="right">677</td>
</tr>
<tr class="odd">
<td>E2_DOX</td>
<td align="right">4398</td>
<td align="right">4483</td>
</tr>
<tr class="even">
<td>Progesterone_DOX</td>
<td align="right">842</td>
<td align="right">614</td>
</tr>
<tr class="odd">
<td>E2_DOX_specific</td>
<td align="right">27</td>
<td align="right">321</td>
</tr>
<tr class="even">
<td>Progesterone_DOX_specific</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
</div>
<div id="genes-that-are-differentially-expressed" class="section level2">
<h2>Genes that are differentially expressed</h2>
<p><em>Include genes with qvalue &lt; 0.05 for each comparison group. You can use the qvalue column to further filter genes based on significance. You can also use LogFC as a filter to select genes that are up- or down-regulated</em></p>
<ul>
<li><a href="170305_E2_PBS.txt"><em>Genes that are differentially regulated by E2 in PBS</em></a></li>
<li><a href="170305_Progesterone_PBS.txt"><em>Genes that are differentially regulated by Progesterone in PBS</em></a></li>
<li><a href="170305_E2_DOX.txt"><em>Genes that are differentially regulated by E2 in DOX</em></a></li>
<li><a href="170305_Progesterone_DOX.txt"><em>Genes that are differentially regulated by Progesterone in DOX</em></a></li>
<li><a href="Compariosn_1.txt"><em>Genes that are differentially specifically regulated by E2 in DOX compared to in PBS</em></a></li>
<li><a href="Comparison_2.txt"><em>Genes that are differentially specifically regulated by Progesterone in DOX compared to in PBS</em></a></li>
</ul>
<hr />
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>## R version 3.2.0 (2015-04-16)
## Platform: x86_64-unknown-linux-gnu (64-bit)
## Running under: Ubuntu 14.04.2 LTS
## 
## locale:
##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] edgeR_3.4.2       limma_3.18.13     data.table_1.10.4
## 
## loaded via a namespace (and not attached):
##  [1] backports_1.0.5 magrittr_1.5    rprojroot_1.2   tools_3.2.0    
##  [5] htmltools_0.3.5 yaml_2.1.14     Rcpp_0.12.9     stringi_1.1.2  
##  [9] rmarkdown_1.3   highr_0.5.1     knitr_1.15.1    stringr_1.2.0  
## [13] digest_0.6.12   evaluate_0.10</code></pre>
<p><strong>Last updated:</strong> 2017-03-06 11:51:45</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
