---
output:
  html_document:
    toc: true
---

<h1 style="color:grey;font-family:arial; position:relative; top:15px;text-align:center;">XIAO<img src="website_materials/research_website_logo.png"  style="float:right;width:200px;height:150px;"></h1>
<br>

***

```
Read in reads count of protein coding genes of the 1st batch mouse mRNA-Seq data. 
the count data were generated by htseq-count and were stored in ../data/Shuo_Xiao_RNAseq_mouse_batch1
Then use EdgeR to analyze differentiall expressed genes
```

```{r, echo=FALSE}
library(data.table)
library(edgeR)
```

## Read in count of each protein coding gene, short examples shown below


```{r,echo=FALSE}
count <- list()
for(i in 1:24){
  file_name <- paste("../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_", i, "/htseq_protein_coding.count", sep = "")
  temp <- fread(file_name, header = FALSE, sep = "\t", stringsAsFactors = FALSE)
  sample_name <- paste("ART_", i, sep = "")
  if(i == 1){
    count$genename <- temp[,1]
    count[[sample_name]] <- temp[,2]
  } else{
    count[[sample_name]] <- temp[,2]
  }
}
count_header <- names(count)
count <- as.data.table(count)
colnames(count) <- count_header
# need to remove the last few lines that count reads for other features. 
count <- count[1:(nrow(count)-5)]
knitr::kable(head(count))
```

```{r, cache=TRUE, dependson=count}
fwrite(count, "./170305_read_count_table_for_protein_coding_genes.txt", col.names = TRUE, sep = "\t", row.names = FALSE, quote = FALSE)
```

* [**Click here for read count information for protein coding genes**](170305_read_count_table_for_protein_coding_genes.txt)

## use EdgeR to call differentially regulated genes

```{r, cache=TRUE, dependson=count, echo=FALSE}
groups <- factor(c(rep("PBS_control", 4), rep("PBS_E2", 4), rep("PBS_pro", 4), rep("DOX_control", 4), rep("DOX_E2", 4), rep("DOX_pro", 4)))

y <- DGEList(counts=count[,-1], genes=count$genename,group=groups)
y <- calcNormFactors(y)
design <- model.matrix(~group,data=y$samples)
y <- estimateGLMCommonDisp(y,design)
y <- estimateGLMTrendedDisp(y,design)
# tag wise dispersion is squeezed towards trended dispersion. i.e., the prior is trended. 
y <- estimateGLMTagwiseDisp(y,design)
fit <- glmFit(y,design)

# then make comparisons

diff_table <- list()

lrt.PBS_E2 <- glmLRT(fit,contrast = c(0,0,0,-1,1,0))
lrt.PBS_E2_table <- data.table(genename = lrt.PBS_E2$genes, lrt.PBS_E2$table)
lrt.PBS_E2_table$qvalue <- p.adjust(lrt.PBS_E2_table$PValue, method = "BH")
diff_table$PBS_E2 <- lrt.PBS_E2_table

lrt.PBS_pro <- glmLRT(fit,contrast = c(0,0,0,-1,0,1))
lrt.PBS_pro_table <- data.table(genename = lrt.PBS_pro$genes, lrt.PBS_pro$table)
lrt.PBS_pro_table$qvalue <- p.adjust(lrt.PBS_pro_table$PValue, method = "BH")
diff_table$PBS_pro <- lrt.PBS_pro_table

lrt.DOX_E2 <- glmLRT(fit,coef = 2)
lrt.DOX_E2_table <- data.table(genename = lrt.DOX_E2$genes, lrt.DOX_E2$table)
lrt.DOX_E2_table$qvalue <- p.adjust(lrt.DOX_E2_table$PValue, method = "BH")
diff_table$DOX_E2 <- lrt.DOX_E2_table

lrt.DOX_pro <- glmLRT(fit,coef = 3)
lrt.DOX_pro_table <- data.table(genename = lrt.DOX_pro$genes, lrt.DOX_pro$table)
lrt.DOX_pro_table$qvalue <- p.adjust(lrt.DOX_pro_table$PValue, method = "BH")
diff_table$DOX_pro <- lrt.DOX_pro_table

# group 5/group 4 vs group 2/group 1, compare E2/control effect with and without DOX
lrt.1 <- glmLRT(fit, contrast = c(0,1,0,1,-1,0))
lrt.1_table <- data.table(genename = lrt.1$genes, lrt.1$table)
lrt.1_table$qvalue <- p.adjust(lrt.1_table$PValue, method = "BH")
diff_table$compare_1 <- lrt.1_table

# group 6/group 4 vs group 3/group 1, compare pro/control effect with and without DOX
lrt.2 <- glmLRT(fit, contrast = c(0,0,1,1,0,-1))
lrt.2_table <- data.table(genename = lrt.2$genes, lrt.2$table)
lrt.2_table$qvalue <- p.adjust(lrt.2_table$PValue, method = "BH")
diff_table$compare_2 <- lrt.2_table
```

## Number of genes up- and down-regulated at qvalue < 0.05
```{r,echo=FALSE}
diff_q_0.05 <- data.frame(up = rep(0,6), down = rep(0,6))
for(i in 1:6){
  diff_q_0.05[i,1] <- nrow(diff_table[[i]][qvalue<0.05 & logFC > 0])
  diff_q_0.05[i,2] <- nrow(diff_table[[i]][qvalue<0.05 & logFC < 0])
}
rownames(diff_q_0.05) <- c("E2_PBS", "Progesterone_PBS", "E2_DOX", "Progesterone_DOX", "E2_DOX_specific", "Progesterone_DOX_specific")
knitr::kable(diff_q_0.05)
```

## Number of genes up- and down-regulated at qvalue < 0.01
```{r,echo=FALSE}
diff_q_0.01 <- data.frame(up = rep(0,6), down = rep(0,6))
for(i in 1:6){
  diff_q_0.01[i,1] <- nrow(diff_table[[i]][qvalue<0.01 & logFC > 0])
  diff_q_0.01[i,2] <- nrow(diff_table[[i]][qvalue<0.01 & logFC < 0])
}
rownames(diff_q_0.01) <- c("E2_PBS", "Progesterone_PBS", "E2_DOX", "Progesterone_DOX", "E2_DOX_specific", "Progesterone_DOX_specific")
knitr::kable(diff_q_0.01)
```

## Number of genes up- and down-regulated at qvalue < 0.001
```{r,echo=FALSE}
diff_q_0.001 <- data.frame(up = rep(0,6), down = rep(0,6))
for(i in 1:6){
  diff_q_0.001[i,1] <- nrow(diff_table[[i]][qvalue<0.001 & logFC > 0])
  diff_q_0.001[i,2] <- nrow(diff_table[[i]][qvalue<0.001 & logFC < 0])
}
rownames(diff_q_0.001) <- c("E2_PBS", "Progesterone_PBS", "E2_DOX", "Progesterone_DOX", "E2_DOX_specific", "Progesterone_DOX_specific")
knitr::kable(diff_q_0.001)
```

## Genes that are differentially expressed
*Include genes with qvalue < 0.05 for each comparison group. You can use the qvalue column to further filter genes based on significance. You can also use LogFC as a filter to select genes that are up- or down-regulated* 

```{r, cache=TRUE, dependson=diff_table, echo=FALSE}
diff_gene_filename <- c("170305_E2_PBS.txt", "170305_Progesterone_PBS.txt", "170305_E2_DOX.txt","170305_Progesterone_DOX.txt","Comparison_1.txt", "Comparison_2.txt")
for(i in 1:6){
fwrite(diff_table[[i]][qvalue<0.05], diff_gene_filename[i], col.names = TRUE, sep = "\t", row.names = FALSE, quote = FALSE )
}
```

* [*Genes that are differentially regulated by E2 in PBS*](170305_E2_PBS.txt)
* [*Genes that are differentially regulated by Progesterone in PBS*](170305_Progesterone_PBS.txt)
* [*Genes that are differentially regulated by E2 in DOX*](170305_E2_DOX.txt)
* [*Genes that are differentially regulated by Progesterone in DOX*](170305_Progesterone_DOX.txt)
* [*Genes that are differentially specifically regulated by E2 in DOX compared to in PBS*](Compariosn_1.txt)
* [*Genes that are differentially specifically regulated by Progesterone in DOX compared to in PBS*](Comparison_2.txt)


***

```{r}
sessionInfo()
```
**Last updated:** `r Sys.time()`