<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 54px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 59px;
  margin-top: -59px;
}

.section h2 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h3 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h4 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h5 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h6 {
  padding-top: 59px;
  margin-top: -59px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">XIAO Project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="meeting.html">Meeting</a>
</li>
<li>
  <a href="Note.html">Note</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<h1 style="color:grey;font-family:arial; position:relative; top:15px;text-align:center;">
Collaboration J project<img src="website_materials/research_website_logo.png"  style="float:right;width:200px;height:150px;">
</h1>
<p><br></p>
<hr />
<pre class="bash"><code>for i in ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_*/
do
echo $i
cat $i/accepted_hits.bam.flagstat | head -3
echo &quot;-------------------------------------------------------------------&quot;
echo &quot;-------------------------------------------------------------------&quot;
done</code></pre>
<pre><code>## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_1/
## 25905640 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 25905640 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_10/
## 20505234 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 20505234 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_11/
## 27476019 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 27476019 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_12/
## 25035951 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 25035951 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_13/
## 27645814 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 27645814 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_14/
## 25218950 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 25218950 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_15/
## 22557516 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 22557516 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_16/
## 26560768 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 26560768 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_17/
## 27170648 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 27170648 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_18/
## 26202638 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 26202638 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_19/
## 28269413 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 28269413 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_2/
## 24853696 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 24853696 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_20/
## 33571741 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 33571741 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_21/
## 26057366 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 26057366 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_22/
## 27877495 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 27877495 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_23/
## 32456353 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 32456353 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_24/
## 32417274 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 32417274 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_3/
## 23238999 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 23238999 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_4/
## 25118738 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 25118738 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_5/
## 47533398 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 47533398 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_6/
## 22562756 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 22562756 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_7/
## 25734808 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 25734808 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_8/
## 24152381 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 24152381 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------
## ../data/Shuo_Xiao_RNAseq_mouse_batch1/ART_9/
## 26069281 + 0 in total (QC-passed reads + QC-failed reads)
## 0 + 0 duplicates
## 26069281 + 0 mapped (100.00%:-nan%)
## -------------------------------------------------------------------
## -------------------------------------------------------------------</code></pre>
<p><strong>Last updated:</strong> 2017-03-03 23:28:43</p>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
